packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/MultiSelectFacetsGroup.tsx

```diff
@@ -56,6 +56,9 @@ export const MultiSelectFacetsGroup: FC<MultiSelectFacetsGroupProps> = ({
     <>
       {facets.map(facet => {
         const text = get(facet, facetsTextField, '');
+        const matchingResults = facet.matching_results;
+        const labelText =
+          matchingResults !== undefined ? text + ' (' + matchingResults + ')' : text;
```

Comment:
This is not translatable. I think best option is to have two different i18n message strings:
- `{facetText}`
- `{facetText} ({count})`

Then choose correct one and substitute values.

Reply:
This should be resolved now